api_products.md

```diff
@@ -11,25 +11,214 @@ information about api access.
+| `fsk`       | Product was rated by the german "Freiwillige Selbstkontrolle der Filmwirtschaft" (FSK)                         |
+
+
 ### Product Object
-A single product is encoded as follows.
+
+A product has two differen representations. The [first `Product Object
+v2`](#product-object-v2) is newer and supports more features. The
+[second `Product Object v1`](#product-object-v1) is deprecated and
+should not be used anymore.
+
+The service use content negoation to distinguish between the
+representation. Please use
+[`Accept`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept)
+and
+[`Content-Type`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type)
+headers with the MIME types named below.
+
+#### Product Object v2
+
+*MIME type*: `application/vnd.snabble.product.v2+json`
+
+Example:
+```
+{
+   "sku" : "1120325205",
+   "project": "project",
+   "name" : "Premium-Holzöl",
+   "description": "farblos, 750ml",
+   "subtitle" : "Aplina",
+   "taxCategory" : "normal",
+   "imageUrl" : "https://www.example.com/some/image.jpg",
+   "productType" : "default",
+   "controlIndication": 0.4,
+   "saleRestriction": "min_age_18",
+   "saleStop": true,
+   "codes": [
+       {
+           "code": "0654203316514",
+           "transmissionCode": "1234203316514"
+           "template": "default"
+       }
+   ]
+}
+```
+
+Product attributes:
+
+| Parameter           | Type       | Default   | Description                                                                                               |
+|---------------------|------------|-----------|-----------------------------------------------------------------------------------------------------------|
+| `sku`               | `string`   |           | The unique id for identification of the product                                                           |
+| `project`           | `string`   |           | The project                                                                                               |
+| `name`              | `string`   | ""        | The display name of the product                                                                           |
+| `description`       | `string`   | null      | A short description of the product                                                                        |
+| `subtitle`          | `string`   | null      | An additional title line for individual use (e.g. brand information)                                      |
+| `imageUrl`          | `string`   | null      | The full URL for a product image                                                                          |
+| `taxCategory`       | `string`   | null      |                                                                                                           |
+| `depositProduct`    | `string`   | null      | The SKU of a corresponding product containing the associated deposit article                              |
```

Comment:
The SKU of the associated deposit article
Ein Produkt enthält ja keine article sondern repräsentiert sie